onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -483,7 +483,7 @@ def t5_to_onnx(args: argparse.Namespace):
         Path(args.output).parent,
         use_gpu=args.use_gpu,
         use_external_data_format=args.use_external_data_format,
-        optimize_onnx=False,
+        optimize_onnx=(args.precision != Precision.FLOAT16),
```

Comment:
Any reason why float16 is not needed optimization?

Reply:
currently there seems to be some issues with fp16 converter. Will revisit it when most of the ops are fused

Reply:
I see.